app-emulation/box64/autobuild/defines

```diff
@@ -0,0 +1,31 @@
+PKGNAME=box64
+PKGSEC=utils
+PKGDES="an userspace x86_64 emulator with a twist, aimed to run x86_64 Linux programs on non-x86_64 Linux systems"
```

Comment:
This is quite strictly against the Styling Manual:

https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#ruan-jian-bao-shuo-ming

- You did not capitalise the first letter.
- You reproduced upstream propaganda (WTF was that "twist"?) - think: does it help the user?
- Have you noticed that people are using Box64 with Wine, too?
- Does Box64 work on x86 systems? I think not.
- Terminlogy - it's x86-64.

How about:

```
Userspace emulator for running x86-64 applications on non-x86 systems
```